velox/docs/monitoring/stats.rst

```diff
@@ -39,3 +40,26 @@ Memory Arbitration
      - kByte
      - The reclaimed memory bytes of an operator during the memory arbitration.
        This stats only applies for spillable operators.
+
+HashBuild, HashAggregation
+--------------------------
+
+.. list-table::
+   :widths: 40 10 50
+   :header-rows: 1
+
+   * - Stats Name
+     - Type
```

Comment:
I'm thinking that all runtime stats values are integers. Some are simple counters, while others have a unit (bytes or nanoseconds). Hence, perhaps, the table header could say "Unit" and the values will be either `<empty>` (default, simple counter), bytes, or nanoseconds.

What do you think?

```
struct RuntimeMetric {
  // Sum, min, max have the same unit, count has kNone.
  RuntimeCounter::Unit unit;
  int64_t sum{0};
  int64_t count{0};
  int64_t min{std::numeric_limits<int64_t>::max()};
  int64_t max{std::numeric_limits<int64_t>::min()};
```

Reply:
Sounds great. Updated.